examples/preprocessing/plot_xdawn_denoising.py

```diff
@@ -68,13 +68,23 @@
 signal_cov = compute_raw_covariance(raw, picks=picks)
 
 # Xdawn instance
-xd = Xdawn(n_components=2, signal_cov=signal_cov)
+xd = XdawnTransformer(n_components=2, signal_cov=signal_cov)
 
-# Fit xdawn
-xd.fit(epochs)
+# Ensure data Independence
+train, test = range(0, len(epochs), 2), range(1, len(epochs), 2)
+
+# Fit xdawn and pass events to correct for overlapping epochs
+X = epochs.get_data()
+y = epochs.events[:, 2]
+xd.fit(X[train], y[train], events=epochs.events[train])
 
 # Denoise epochs
-epochs_denoised = xd.apply(epochs)
+Xt = xd.transform(X[test])
+
+# Plot in sensor space
+X_denoised = xd.inverse_transform(Xt)
+epochs_denoised = epochs.copy()
+epochs_denoised._data[test] = X_denoised
```

Comment:
yes. I'm actually wondering whether I shouldn't just plot the data, since we're not using any particular topography, might as well keep it simple, WDYT?